frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -98,6 +98,14 @@ class EntityMenu extends Component {
 
                 const key = item.key ?? item.title;
 
+                if (item.separator) {
+                  return (
+                    <li key={key}>
+                      <Divider m="sm" />
+                    </li>
+                  );
+                }
+
```

Comment:
we shouldn't be extending this component. When we touch this dropdown, we should use `Menu` from `Metabase/ui`